hack/config-default.sh

```diff
@@ -43,6 +43,8 @@ conn_check_dns=${CONN_CHECK_DNS:-""}
 migration_network_nic=${MIGRATION_NETWORK_NIC:-"eth1"}
 infra_replicas=${KUBEVIRT_INFRA_REPLICAS:-0}
 test_image_replicas=${KUBEVIRT_E2E_PARALLEL_NODES:-6}
+base_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
```

Comment:
also would appreciate to know why you needed to move this here

Reply:
Otherwise `kubevirt_test_config` is a private in functests.sh so we don't have access to it